### ISSUE TITLE
Empty table header usage across the admin

### ISSUE THREAD

**Issue by lb-** [wagtail/wagtail#11596]
### Issue Summary

As a follow on from https://github.com/wagtail/wagtail/issues/11459 - it's been found that there are other usages of empty `th` elements used for visual layout only.

These should be updated using a similar approach to https://github.com/wagtail/wagtail/pull/11465

### Details

1. https://github.com/wagtail/wagtail/blob/c99c74cc7e5f7d534fecb6ab0be5a9c7c4da682a/client/src/entrypoints/contrib/typed_table_block/typed_table_block.js#L79
2. https://github.com/wagtail/wagtail/blob/c99c74cc7e5f7d534fecb6ab0be5a9c7c4da682a/wagtail/admin/templates/wagtailadmin/permissions/includes/collection_member_permissions_formset.html#L29
3. https://github.com/wagtail/wagtail/blob/c99c74cc7e5f7d534fecb6ab0be5a9c7c4da682a/wagtail/admin/templates/wagtailadmin/workflows/includes/workflow_pages_formset.html#L20
4. https://github.com/wagtail/wagtail/blob/c99c74cc7e5f7d534fecb6ab0be5a9c7c4da682a/wagtail/admin/ui/tables/pages.py#L75
5. https://github.com/wagtail/wagtail/blob/c99c74cc7e5f7d534fecb6ab0be5a9c7c4da682a/wagtail/users/templates/wagtailusers/groups/includes/formatted_permissions.html#L123
6. https://github.com/wagtail/wagtail/blob/c99c74cc7e5f7d534fecb6ab0be5a9c7c4da682a/wagtail/users/templates/wagtailusers/groups/includes/page_permissions_formset.html#L28

As for the `<th></th>`, it appears that some are in JavaScript, if updating these be sure to update the Jest snapshots (`npm run test:unit -- -u`) but also ensure that the JS code does not 'fill these in' with titles somehow first.

wagtail/admin/static_src/wagtailadmin/js/vendor/jquery.datetimepicker.js (ignore this file, it's going to be removed in the near future)

_Originally posted by @lb- in https://github.com/wagtail/wagtail/issues/11459#issuecomment-1902079223_
            

### Working on this

Anyone can contribute to this. View our [contributing guidelines](https://docs.wagtail.org/en/latest/contributing/index.html), add a comment to the issue once you’re ready to start.

**Comment by Jaivignesh-afk** [comment]
I am working on this

**Comment by Jaivignesh-afk** [comment]
May I ask for further clarification? Should I utilize `aria-hidden=true` or `role=presentation`? Additionally, Is it ok to remove empty headers that are not listed above?

**Comment by lb-** [comment]
@Jaivignesh-afk  let's go with aria-hidden=true as that's what was done in the previous PR
https://github.com/wagtail/wagtail/pull/11465

If you find other candidates for empty elements that need fixing this same way, feel free to include in your PR (please add screenshots so we know where to find them).

Be careful on simply removing them though, they may be in place to aid the visual layout.

**Comment by Jaivignesh-afk** [comment]
ok

**Comment by lb-** [comment]
Looks like a few of these issues were fixed via https://github.com/wagtail/wagtail/pull/12203
